src/otx/algo/classification/backbones/vision_transformer.py

```diff
@@ -291,6 +292,25 @@ def __init__(  # noqa: PLR0913
 
         self.norm = norm_layer(embed_dim)
 
+        self.lora = lora
+        if self.lora:
+            lora_rank = 8
+            lora_alpha = 1.0
```

Comment:
I will perform more experiments on LoRA parameters. At this moment, because of some performance degradation on ViT-Tiny, I am thinking about employing freezing some blocks of transformer in terms of partial training. Anyway, I will get back to you :)